lib/config/index.js

```diff
@@ -12,4 +30,7 @@ nconf.argv()
     });
 
 nconf.defaults(defaults);
-module.exports = nconf;
\ No newline at end of file
+
+// @TODO: this file gets cached after the first require, we need to redesign how we load the config
+// @TODO: for example: require('ghost-ignition').config()
```

Comment:
I have a feeling that this is a bigger issue with:

1. the approach used in this file, of making assumptions about where config will be (not just the default config but also on L29
2. the use of ignition inside of gscan

Where we're seeing this file cause an issue is where this is used inside a module which is then a submodule of a different project. In this case, we can't assume where the default config, or main config lives? I think we can assume process.cwd() as a default, but also maybe need to be able to pass in a path?

In the real-world case of gscan, this probably shouldn't have been done this way, as the configuration is only used for the website-serving part. Trying to use one module for both the checking utility and the website is probably where this is falling down?